### PR TITLE
fix: Declarative shadow-dom error in Chrome

### PR DESCRIPTION
**Pull request by ethanWallace**
# Summary | Résumé

Prevent errors from appearing in Chrome by adjusting render logic.

## Changes

- Added a list of our components that are form-associated to add `shadowrootdelegatesfocus` to template tag in DSD
- Fixed render function from attaching multiple template tags to custom elements in components

### CODE REVIEW

File: packages/web/stencil.config.ts [hunk 1]
```diff
@@ -51,5 +51,6 @@ export const config: Config = {
   validatePrimaryPackageOutputTarget: true,
   extras: {
     enableImportInjection: true,
+    experimentalSlotFixes: true
```
Comment (daine) [inline packages/web/stencil.config.ts:54]: Is there a way for us to guard for things like these that are experimental?

**Review by daine — COMMENTED** [review]
I wonder if we should start working on some e2e tests to help test and review as well?

File: packages/web/stencil.config.ts (hunk 1 above)
Comment (ethanWallace) [reply packages/web/stencil.config.ts:54]: This addition doesn't directly fix the issue this PR addresses. I just decided to add it into our build to hopefully protect against any slot issues.

**Review by melaniebmn — APPROVED** [review]
LGTM

**Pull request merged**